lib/ecto/adapters/clickhouse/structure.ex

```diff
@@ -96,7 +94,7 @@ defmodule Ecto.Adapters.ClickHouse.Structure do
 
   defp run_with_cmd(cmd, cmd_args, opts) do
     args = ["--host", opts[:hostname] || "localhost"]
-    args = if username = opts[:username], do: ["--username", username | args], else: args
+    args = if username = opts[:username], do: ["--user", username | args], else: args
```

Comment:
https://github.com/plausible/ecto_ch/pull/111 probably obsoletes this change

Reply:
Indeed, so nothing really remains here 🙂